cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -83,4 +83,14 @@ function up() {
             exit 1
         fi
     fi
+
+    if [ "$KUBEVIRT_DEPLOY_CDI" == "true" ]; then
```

Comment:
since its a boolean, no need quotes here

Reply:
IIRC bash doesn't have the concept of boolean itself when declaring variables, no?

Reply:
not sure, as it will work also on other strings indeed